projects/shared-components/cookie-consent/cookie-consent.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+
```

Comment:
We're on the right path, you can even replicate what functionality other web apps have for the cookie consent. Let's make it a simple and cool one

Reply:
Ok, will work on that.